src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableEntityContext.cs

```diff
@@ -415,12 +417,94 @@ string IDurableEntityContext.StartNewOrchestration(string functionName, object i
             return instanceId;
         }
 
+        async Task IDurableEntityContext.DispatchAsync<T>(params object[] constructorParameters)
```

Comment:
Should there be a `\\\ <inheritdoc/>` (inherit documentation) comment over this?

Reply:
Actually, the way this is implemented means that `<inheritdoc/>` isn't useful.

Essentially we implemented these interface methods explicitly, meaning that if someone has the object cast as `DurableEntityContext`, they can't actually call this method. They have to first cast it to `IDurableEntityContext` before they use the method, meaning the IDE will already be providing the docs.

I'm not in love with explicit interface method implementations, but they are spread throughout the codebase, so I keep it this way for consistency.